<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите колличество вершин = </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="kolV">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>3</number>
    </property>
   </widget>
   <widget class="QPushButton" name="MatrBut">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Задать</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="MatrA">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>921</width>
      <height>551</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>47</width>
      <height>51</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>921</width>
      <height>551</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>A=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Error">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>error</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBut">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResBut">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>660</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Расчет графа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DelBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>660</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить Вершину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddBut">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>660</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить Вершину</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
